fastlane/CommonFastfile

```diff
@@ -142,11 +146,43 @@ lane :github_changelog do |options|
       item = items.first
       message = "#{item['title']} (##{item['number']})"
       username = item["user"]["login"]
+      change_types = item["labels"].map do |label|
+        label_name = label["name"]
+        supported_types.include? label_name || "other"
```

Comment:
This might be wip, and I might be wrong, but I think this would map the labels to booleans? Since the last line within the `map` returns a boolean.

Another thing is that I prefer to use parentheses for most of these, and I think it's recommended according to rubocop.

Reply:
Oh I think this was working and I broke it in one of the last commits 🙈

Reply:
fixed